### ISSUE TITLE
Issue with connectivity

### ISSUE THREAD

**Issue by BilliTheKid** [dride/dride-app-issues#42]
With the implementation of the red banner to connect to Dride it's causing issues with auto pairing in Android.

**Comment by saoron** [comment]
Can you upload a video?

**Comment by BilliTheKid** [comment]
@saoron Yes but that's not the issue. The issue is connectivity from the app to the unit. With having to press the "connect to Dride red banner" there's an issue with auto connection. The red banner connect option should be removed.

**Comment by saoron** [comment]
I mean upload a video here so I can see the connection process

**Comment by BilliTheKid** [comment]
@saoron I will do one later on

**Comment by BilliTheKid** [comment]
@saoron It knocked off twice while driving today. So when I turned on the engine it took around 50 secs to 1 minute to connect which is very slow. Then after 5 mins the WiFi symbol went and didn't appear back until I opened the app (which I shouldn't have to) I'm not sure if this is an issue with the red banner connect button that has been implemented into the Android app but there is definitely a bug with connectivity.
